Data/future/cbot/margins/EH.csv

```diff
@@ -1,61 +1,193 @@
 date,initial,maintenance
-# https://www.cmegroup.com/tools-information/lookups/advisories/clearing/Chadv12-019.html
 20120117,4725,3500
-# https://www.cmegroup.com/tools-information/lookups/advisories/clearing/Chadv12-221.html
 20120529,4050,3000
-# https://www.cmegroup.com/tools-information/lookups/advisories/clearing/Chadv12-328.html
 20120806,4455,3300
-# https://www.cmegroup.com/tools-information/lookups/advisories/clearing/Chadv13-168.html
+20130102,3630,3300
+20130212,3080,2800
 20130409,5130,3800
-# https://www.cmegroup.com/tools-information/lookups/advisories/clearing/Chadv13-594.html
+20130410,4180,3800
+20131202,2970,2700
+20131204,4180,3800
+20140102,2970,2700
 20140103,5940,4400
-# https://www.cmegroup.com/tools-information/lookups/advisories/clearing/Chadv14-114.html
+20140106,4840,4400
+20140203,2970,2700
+20140205,4840,4400
+20140303,2970,2700
+20140305,4840,4400
```

Comment:
This back and forth over 2 days seems unlikely - please sanity check the data. Possibly it is reading the margins of two different contracts and being mapped to this one.